pkg/rpc/client/rpc.go

```diff
@@ -150,6 +302,61 @@ func (c *Client) GetRawTransactionVerbose(hash util.Uint256) (*result.Transactio
 	return resp, nil
 }
 
+// GetStorage returns the stored value, according to the contract script hash and the stored key.
+func (c *Client) GetStorage(hash util.Uint160, key []byte) (*state.StorageItem, error) {
```

Comment:
If we're not getting `IsConst` from the server, then maybe it's not worth creating a `state.StorageItem` wrapper, you may just return `[]byte`.